doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/10_V6_to_V10.md

```diff
@@ -1,12 +1,57 @@
-# Upgrading Pimcore from Version 6.9 to Version 10
+# Upgrading Pimcore from Version 6.x to Version 10
 
-# Flush your `/tmp` directories
+## System Requirements
+ - PHP >= 8.0
+ - Apache >= 2.4
+
+## Database
+- MariaDB >= 10.3
+- MySQL >= 8.0
+- Percona Server (supported versions see MySQL)
+
+## Preparatory Work
+- If you're on a version between 6.0.0 and 6.8.x, then please update to 6.9.x first and then continue with the upgrade steps.
+
+## IMPORTANT CHANGES PRIOR THE UPDATE!
+
+### Migrating legacy module/controller/action configurations to new controller references
+You can use `./bin/console migration:controller-reference` to migrate your existing Documents, Static routes and Document Types to the new controller references in the format: `AppBundle\Controller\FooController::barAction`.
+
+### Cleanup your composer.json
+Update composer.json as per [skeleton](https://github.com/pimcore/skeleton/blob/master/composer.json)
+
+### Update folder structure 
+Update your project folder structure as per [Symfony Flex](https://symfony.com/doc/current/setup/flex.html) guidelines. See demo changes [here](https://github.com/pimcore/demo/pull/203/files)
+
+Important changes:
+- Move configs from app/config/*.yml to config/*.yaml
+- Move templates from app/Resources/views to templates and bundles templates from 
```

Comment:
Something is missing here...

Reply:
I think it is clearer if it is rewritten and formatted:

> - Move templates from bundles and `app/Resources/views` to `templates`

Reply:
done!